pub.go

```diff
@@ -11,12 +11,6 @@ import (
 	"sync"
 )
 
-// Topics is an interface that wraps the basic Topics method.
```

Comment:
why do you remove it altogether?
(instead of just moving it to `sub.go`?)

it seems to me it's still a rather interesting interface (one that `Sub` still implements in your new scheme).
WDYT?

Reply:
Makes sense. Moved to sub.go.